`build-security-analytics-plugin` stage failed while generating packages

## Description

While analyzing https://github.com/wazuh/wazuh-qa-automation/issues/5385, an error in the `build-security-analytics-plugin` stage was detected while generating indexer packages:

```r
Caused by: org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find com.wazuh:wazuh-indexer-common:5.0.0.latest.
Searched in the following locations:
  - https://repo.maven.apache.org/maven2/com/wazuh/wazuh-indexer-common/5.0.0.latest/wazuh-indexer-common-5.0.0.latest.pom
  - file:/home/runner/.m2/repository/com/wazuh/wazuh-indexer-common/5.0.0.latest/wazuh-indexer-common-5.0.0.latest.pom
  - https://ci.opensearch.org/ci/dbc/snapshots/maven/com/wazuh/wazuh-indexer-common/5.0.0.latest/wazuh-indexer-common-5.0.0.latest.pom
Required by:
    root project :


BUILD FAILED in 21s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.14.3/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
Error: Process completed with exit code 1.
```

## Evidence

- indexer-rpm-aarch64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20939483538/job/60169772279
- indexer-deb-arm64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20939482670/job/60169765040
- indexer-rpm-x86_64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20939483094/job/60169773553
- indexer-deb-amd64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/20939482207/job/60169766169

--- a/.github/workflows/5_builderpackage_indexer.yml
+++ b/.github/workflows/5_builderpackage_indexer.yml
@@ -393,6 +393,13 @@ jobs:
         id: version
         run: echo "version=$(jq .version -r VERSION.json)" >> "$GITHUB_OUTPUT"
 
+      - name: Publish commons library to mavenLocal
+        run: |
+          cd commons
+          ../gradlew publishToMavenLocal -x check \
+            -Dversion=${{ steps.version.outputs.version }} \
+            -Drevision=${{ inputs.revision }}
+
       - name: Build with Gradle
         run: ./gradlew build -x integTest -Dversion=${{ steps.version.outputs.version }} -Drevision=${{ inputs.revision }}
 

